models/bsv/fee_unit.go

```diff
@@ -0,0 +1,32 @@
+package bsv
+
+import "fmt"
+
+// FeeUnit displays the amount of Satoshis neededz
+// for a specific amount of Bytes in a transaction
+// see https://github.com/bitcoin-sv-specs/brfc-misc/tree/master/feespec
+// Imported from deprecated go-bt library
+type FeeUnit struct {
+	Satoshis int `json:"satoshis"` // Fee in satoshis of the amount of Bytes
+	Bytes    int `json:"bytes"`    // Number of bytes that the Fee covers
+}
```

Comment:
I'm also defining the `FeeUnit` in my PR (https://github.com/bitcoin-sv/spv-wallet/pull/737)
hmmm...
Another thing to discuss 🙈